packages/angular-cli/commands/init.run.ts

```diff
@@ -31,9 +34,11 @@ export default function initRun(commandOptions: any, rawArgs: string[]) {
 
   let npmInstall: any;
   if (!commandOptions.skipNpm) {
+    const packageManager = CliConfig.fromProject().get('packageManager') || 'npm';
```

Comment:
No need to `||` for values in the config, you should set a `default` value in the schema.json instead.